src/App.js

```diff
@@ -16,23 +16,23 @@ function App() {
   };
 
   // Implement functions to add data to contacts and appointments
-  const addContact = (contactName, phone, email) => {
-    let newContact = {
+  const addContacts = (contactName, phone, email) => {

```

Comment:
@Any22 please keep this method addContact singular, as it is only adding one new contact, not multiple contacts.